quick-start.md

```diff
@@ -105,3 +107,196 @@ sudo su
+    * Google Cloud Kubernetes: [Ingress with NGINX controller on Google Kubernetes Engine](https://kubernetes.github.io/ingress-nginx/deploy/)
+    * AWS EKS: [Install NGINX Controller for AWS EKS](https://kubernetes.github.io/ingress-nginx/deploy/#network-load-balancer-nlb)
+    * Microk8s: [Add on: Ingress](https://microk8s.io/docs/addon-ingress)
+* Script tested on Minikube with Kubernetes v1.18.0
+
+### Deployment Steps
+
+1. Fetch the **install.k8s.sh** script on the system you want to deploy appsmith
+
+```bash
+# Downloads install.sh
+curl -O https://raw.githubusercontent.com/appsmithorg/appsmith/master/deploy/k8s/install.k8s.sh
+```
+
+2. Make the script executable
+
+```bash
+chmod +x install.k8s.sh
+```
+
+3. Run the script.
+
+```bash
+./install.k8s.sh
+```
+
+4. Check if all the pods are running correctly.
+
+```bash
+kubectl get pods
+
+#Output should look like this
+NAME                                        READY   STATUS      RESTARTS    AGE
+appsmith-editor-cbf5956c4-2zxlz             1/1     Running     0           4m26s
+appsmith-internal-server-d5d555dbc-qddmb.   1/1     Running     2           4m22s
+imago-1602817200-g28b2                      1/1     Running     0           4m39s
+mongo-statefulset-0                         1/1     Running     0           4m13s
+redis-statefulset-0                         1/1     Running     0           4m00s
+```
+
+5. Custom Appsmith's Configuration
+  * After you successfully run the script, all the configuration files have been downloaded and & stored into `<Installation Path>`
+  * If you want to update your app settings (ex: database host). Go to the `<Installation Path>/config-template`, update the corresponding value in the configmap file, then restart the pods.
+  * Below steps will help you update database hostname of your application:
+    * Open file `appsmith-configmap.yaml` in `<Installation Path>/config-template` folder
+    * Update the value of variable `APPSMITH_MONGODB_URI` to your database host name
+    * Run commands:
+```
+kubectl apply -f appsmith-configmap.yaml
+kubectl scale deployment appsmith-internal-server --replicas=0
+kubectl scale deployment appsmith-internal-server --replicas=1
+```
+
+{% hint style="success" %}
+* You can access the running application on the **Ingress Endpoint** if you not chose to provide custom domain for your application .
+```
+kubectl get ingress
+NAME               CLASS    HOSTS   ADDRESS          PORTS   AGE
+appsmith-ingress   <none>   *       XXX.XXX.XX.XXX   80      2m
+```
+* You may need to wait 2-3 minutes before accessing the application to allow application start (depends on your cluster).
+{% endhint %}
+
+### Custom Domains
+
+To host Appsmith on a custom domain, you can contact your domain registrar and update your DNS records. Most domain registrars have documentation on how you can do this yourself.
```

Comment:
I haven't still understood what hosting Appsmith on custom domain really means - what is the output, or what difference does the user see. I think we should explain that.

Then we should go ahead and explain how to do it in k8s. And at the end a few sentences that help user check if they have configured it all correctly.